extension/package.json

```diff
@@ -1,21 +1,21 @@
 {
-  "name": "sparkmonitor-js",
+  "name": "jupyterlab_sparkmonitor",
```

Comment:
In my email exchange with the original author, he mentioned he's cool with a new/separate repo specifically for the lab extension since the notebook and lab extensions are not cross-compatible. In keeping with that, I renamed the extension to follow jupyterlab naming guidelines